src/dom.d.ts

```diff
@@ -775,6 +775,8 @@ export interface DOMAttributes<Target extends EventTarget>
 	// UI Events
 	onScroll?: UIEventHandler<Target> | undefined;
 	onScrollEnd?: UIEventHandler<Target> | undefined;
+	onScrollSnapChange?: UIEventHandler<Target> | undefined;
+	onScrollSnapChanging?: UIEventHandler<Target> | undefined;
```

Comment:
Actually, are these the correct handlers? Looks like this is meant to be a [`SnapEvent`](https://developer.mozilla.org/en-US/docs/Web/API/SnapEvent), not a `UIEvent`